.github/workflows/wcag_test.yml

```diff
@@ -0,0 +1,24 @@
+name: WCAG tests
+
+on:
+  push:
+    branches: [develop, hotfix/*, support/*]
+  pull_request:
+    branches: [develop, hotfix/*, support/*]
```

Comment:
May as well throw `release/*` in here too. That way we aren't shocked to find a bunch of WCAG issues when we try and merge the `release` branch into `develop`.